src/main/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherDefaultDueDateServiceImpl.java

```diff
@@ -20,9 +20,8 @@ public class CuDisbursementVoucherDefaultDueDateServiceImpl implements CuDisburs
 
     @Override
     public Date findDefaultDueDate() {
-        Calendar calendar = dateTimeService.getCurrentCalendar();
-        calendar.add(Calendar.DAY_OF_MONTH, findNumberOfDaysInTheFuture());
-        return new Date(calendar.getTimeInMillis());
+        LocalDate calculatedDefaultDvDueDate = dateTimeService.getLocalDateNow().plusDays(findNumberOfDaysInTheFuture());
+        return dateTimeService.getSqlDate(calculatedDefaultDvDueDate);
```

Comment:
If you want `getLocalDateNow()` to return a pre-defined date but you want `getSqlDate()` to use the service's actual code, you could set up the dateTimeService as a "spied" object. First create the dateTimeService using an actual instance, then use `Mockito.spy()` to create a proxy that will delegate to the underlying object by default. You can then use one of the `Mockito.doXXX()` methods to override the behavior of individual service methods. (In most cases, don't use `Mockito.when()` to configure a "spied" object, because doing it that way will cause the setup process to invoke the actual method on the wrapped instance.)

For an example of how to set this up, see the `ConcurRequestV4ServiceImplTest` class. It creates a "spied" dateTimeService instance using our custom `TestDateTimeServiceImpl` class for the wrapped object.